pipeline/runtime/iacm_nudge.go

```diff
@@ -0,0 +1,140 @@
+package runtime
+
+import (
+	"errors"
+	"github.com/harness/lite-engine/logstream"
+)
+
+func getIACMNudges() []logstream.Nudge {
+	return []logstream.Nudge{
+		logstream.NewNudge("[ERROR] approval step has been rejected",
+			"Check why the approval step was rejected",
+			errors.New("the approval step has been rejected")),
+		logstream.NewNudge("[ERROR] Error parsing the init_option block",
```

Comment:
@nassergonzalez can you please check with support what's the common way to ask users to contact Harness? we should be consistent with the rest of the platform